notebooks/working/investigating_deconfounder.py

```diff
@@ -389,28 +371,36 @@
 results_df_deconf
 
 # ## Adding random noise to the confounder and seeing how it affects the coefficients on the other variables
```

Comment:
In this section, can you produce a plot where the x-axis is the standard deviation of the noise that is added to the confounder and the y-axis is the estimated coefficient value on the new variable (plus error bars coming from the confidence interval around the coefficient)? Also on the plot should be a horizontal line at the true coefficient value (7 in this case).

I think it will be helpful to show the estimation results breaking down with more noise to see roughly how much noise can be added before we stop getting reasonable estimates.

When describing how much noise can be added before the estimation breaks down, it may be helpful to express it as a percentage `(std_dev_noise / std_dev_on_confounder) * 100`.